rhizome/host/bin/download-boot-image

```diff
@@ -0,0 +1,14 @@
+#!/bin/env ruby
+# frozen_string_literal: true
+
+unless (boot_image = ARGV.shift)
+  puts "need boot_image as argument"
```

Comment:
We can use fail and remove exit 1.

Reply:
The other scripts in rhizome's bin directory use the same argument parsing logic. I prefer to maintain consistency. If we decide to refactor, we should modify all of them.

We might consider using `﻿exit` instead of `﻿fail`, as `﻿fail` also prints the stack trace.